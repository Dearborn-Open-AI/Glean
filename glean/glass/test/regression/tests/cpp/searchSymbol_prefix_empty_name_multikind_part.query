action: searchSymbol
args: {"name": "", "repo_name": "test", "language": [1], "kinds": [7,14,12,8,9], "options": {"detailedResults": false, "exactMatch": false, "ignoreCase": false, "namespaceSearch": false } }
