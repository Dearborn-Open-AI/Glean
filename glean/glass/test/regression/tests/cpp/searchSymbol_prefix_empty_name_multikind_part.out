[
    "@generated",
    {
        "symbolDetails": [],
        "symbols": [
            {
                "kind": 12,
                "language": 1,
                "location": {
                    "filepath": "test.cpp",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 27,
                        "lineBegin": 89,
                        "lineEnd": 89
                    },
                    "repository": "test"
                },
                "name": "Global",
                "symbol": "test/cpp//Global"
            },
            {
                "kind": 12,
                "language": 1,
                "location": {
                    "filepath": "test.cpp",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 41,
                        "lineBegin": 90,
                        "lineEnd": 90
                    },
                    "repository": "test"
                },
                "name": "GlobalClass",
                "symbol": "test/cpp//GlobalClass"
            },
            {
                "kind": 12,
                "language": 1,
                "location": {
                    "filepath": "test.cpp",
                    "range": {
                        "columnBegin": 3,
                        "columnEnd": 26,
                        "lineBegin": 83,
                        "lineEnd": 83
                    },
                    "repository": "test"
                },
                "name": "InU",
                "symbol": "test/cpp//baz/U/InU"
            },
            {
                "kind": 12,
                "language": 1,
                "location": {
                    "filepath": "test.cpp",
                    "range": {
                        "columnBegin": 3,
                        "columnEnd": 40,
                        "lineBegin": 84,
                        "lineEnd": 84
                    },
                    "repository": "test"
                },
                "name": "InUClass",
                "symbol": "test/cpp//baz/U/InUClass"
            },
            {
                "kind": 14,
                "language": 1,
                "location": {
                    "filepath": "test.cpp",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 22,
                        "lineEnd": 24
                    },
                    "repository": "test"
                },
                "name": "foo",
                "symbol": "test/cpp//foo/foo"
            },
            {
                "kind": 14,
                "language": 1,
                "location": {
                    "filepath": "test.cpp",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 42,
                        "lineEnd": 44
                    },
                    "repository": "test"
                },
                "name": "fooFn",
                "symbol": "test/cpp//foo/fooFn"
            },
            {
                "kind": 14,
                "language": 1,
                "location": {
                    "filepath": "test.cpp",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 51,
                        "lineEnd": 59
                    },
                    "repository": "test"
                },
                "name": "h",
                "symbol": "test/cpp//h"
            },
            {
                "kind": 12,
                "language": 1,
                "location": {
                    "filepath": "test.cpp",
                    "range": {
                        "columnBegin": 3,
                        "columnEnd": 32,
                        "lineBegin": 67,
                        "lineEnd": 67
                    },
                    "repository": "test"
                },
                "name": "_secret",
                "symbol": "test/cpp//maybe/Nothing/_secret"
            }
        ]
    }
]